examples/quickstart/client/module_bindings/Message.cs

```diff
@@ -43,9 +43,7 @@ public static SpacetimeDB.SATS.AlgebraicType GetAlgebraicType()
 
 		public static explicit operator Message(SpacetimeDB.SATS.AlgebraicValue value)
 		{
-			if (value == null) {
-				return null;
-			}
```

Comment:
I know its somewhat controvertial but our style guide typically requires the braces here

Reply:
I think it got changed in one of the previous PRs, it's just that module bindings weren't regenerated. If we want to revert, probably worth doing outside of this PR.

Reply:
Ah, I'm actually going to remove that function altogether in the next PR, so hopefully that solves it too.